docs/wiki/GettingStarted.md

```diff
@@ -181,7 +181,7 @@ Last but not least, instead of fetching your own copy of the repository you can
 
 If you are forking or cloning the repository, you can use 'tokens' inside your parameter files. Tokens allow you to test deploying modules in your own environment (i.e. using tokens for your naming conventions), or apply other customizations to your resources (i.e. using your own subscription ID inside a Resource ID string). See details in the [Parameter File Tokens Design](./ParameterFileTokens.md).
 
-The repository contains a [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) that has a default token called `namePrefix`. You can use a custom value here and then start adding the tokens in your parameter files. The tokens default format is `<<tokenName>>` (i.e. `<<namePrefix>>`). Here is an example token:
+The repository contains a [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) that has a default token called `namePrefix`. You can use a custom value here and then start adding the tokens in your parameter files. The tokens default format is `<<tokenName>>` (i.e. `sxx`). Here is an example token:
```

Comment:
I'm not sure I get the example. Would the example not be something like `<<defaultprefix>>`?

Reply:
I've made it clearer in this section. Please check